### ISSUE TITLE
Textfield obscured by keyboard on IOS11

### ISSUE THREAD

**Issue by idsecurity** [peter-iakovlev/Telegram#179]
Hello
Using Telegram on iPhone 6S with IOS 11.0.3. Quite often, but not always, the textfield in the chat window is obscured by the keyboard so that I can't see what I'm typing and can't send the message unless I tap on Chats to get back out to the Chats list and then tap on the active chat again. Then the textfield appears again and I can send the message.

Probably not an exclusive Telegram issue since I have the same problem in iMessage.

Has anybody else noticed this?

Thanks.

**Comment by ehsanshb** [comment]
Yep I have same issues. Specially when I reply to a message , keyboard covered the text field and I could not see what I am typing. I forced to close telegram from recent Apps and then reopen it to solve the problem. I didn't have this issue in other Apps.

**Comment by vkedwardli** [comment]
Still not fixed in v4.6, here's a demo video: https://twitter.com/vkedwardli/status/941300713279258624
